src/main/java/service/Main.java

```diff
@@ -0,0 +1,59 @@
+package service;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import model.FruitTransaction;
+import service.dao.ParserImpl;
+import service.dao.ParserTo;
+import service.dao.ParserToImpl;
+import service.dao.ReadFile;
+import service.dao.ReadFileImpl;
+import service.dao.WriteFile;
+import service.dao.WriteFileImpl;
+import service.storage.Balance;
+import service.storage.PerformingOperation;
+import service.storage.Purchase;
+import service.storage.Return;
+import service.storage.Supply;
+
+public class Main {
+
+    public static void main(String[] args) {
+        final String balanceCode = "b";
+        final String supplyCode = "s";
+        final String purchaseCode = "p";
+        final String returnCode = "r";
+        // Read data from file
+        String filePath = "src/main/resources/fileFrom.csv";
+        File file = new File(filePath);
```

Comment:
Please move this logic to file reader so it's method accept only path to file as String argument